src/firewall-cmd.in

```diff
@@ -61,6 +61,8 @@ Status Options
   --complete-reload    Reload firewall and lose state information
   --runtime-to-permanent
                        Create permanent from runtime configuration
+  --reset-to-defaults
+                       Reset configuration to firewalld version's default configuration
```

Comment:
Leave `--reset-to-defaults` in general? additionally, should it be `--reset-to-defaults` or `--reset-defaults`?